src/moldybreadpkg/fedora.nim

```diff
@@ -365,13 +365,17 @@ method populate_results*(this: FedoraRequest): seq[string] {. base .} =
   while token.len > 0:
     try:
       stdout.write("->")
-      response = this.client.getContent(request)
-      new_pids = this.grab_pids(response)
-      for pid in new_pids:
-        result.add(pid)
-      token = this.get_token(response)
-      request = fmt"{base_request}&sessionToken={token}"
-      stdout.flushFile()
+      response = this.client.request(request, httpMethod = HttpGet)
+      if response.status == "200 OK":
+        notice(fmt"Successfully retrieved populate_results {request}.")
+        new_pids = this.grab_pids(response.body)
+        for pid in new_pids:
+          result.add(pid)
+        token = this.get_token(response.body)
+        request = fmt"{base_request}&sessionToken={token}"
+        stdout.flushFile()
+      else:
+        error(fmt"{response.status}: Method populate_results failed on {request}.")
```

Comment:
Maybe fatal this since we aren't retrying yet? `fatal(fmt"{response.status}: Method populate_results failed on {request}.")`

Reply:
Or, do you want to try to get retries in here as well before fataling?

Reply:
I think the fatal is a good solution for now. Let's wait to add wait/sleep globally from the config.yml. Otherwise we are just adding some more mess.